spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/jdbc/DataSourceHealthContributorAutoConfiguration.java

```diff
@@ -64,6 +65,9 @@
 public class DataSourceHealthContributorAutoConfiguration extends
 		CompositeHealthContributorConfiguration<AbstractHealthIndicator, DataSource> implements InitializingBean {
 
+	@Value("${management.health.db.ignore-routing-datasources:false}")
+	private boolean ignoreRoutingDataSources;
```

Comment:
Rather than using `@Value` field injection, we prefer to use a `@ConfigurationProperties` class enabled with `@EnableConfigurationProperties` similar to how `DiskSpaceHealthIndicatorProperties` is defined and used. In this case it would be called something like `DataSourceHealthIndicatorProperties`. It would have the prefix `management.health.db` and a single property, `ignoreRoutingDataSources`.

Reply:
I´ll do this change. Sorry, I should have been aware of this.

Reply:
Change done.